README.md

```diff
@@ -1,24 +1,218 @@
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://pocolee.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📷 Video Recording <a name="video-record"></a>
+
+- [Video Recoding me explaining the project](https://www.loom.com/share/294e7c26973543829208779c24397c1b)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- A Code editor such as `VsCode`.
+- A web browser.
+- Postgresql, ruby and ruby on rails should be installed on your local machine
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd my-folder
+  git clone https://github.com/arabiu033/budget_app.git
+```
+
+
+### Install
+
+Install this project with:
+
+```sh
+  cd Budget-app
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+
+### Run tests
+
+To run tests, run the following command:
+
+
+```sh
+  rspec spec/
+```
+
+### Deployment <a name="triangular_flag_on_post-deployment"></a>
+
+You can deploy this project using: [Render](https://render.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Aminu Rabiu**
+
+- GitHub: [@arabiu033](https://github.com/arabiu033)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/larabiu033)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add categories icon:** user can choose a category icon from a list of options.
+- [ ] **Add confirmation functionality:** newly registered users can receive an email to confirm their registration.
+- [ ] **Sending email for resetting user password**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please help it stand out by giving it a star ✨
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- I would like to thank Microvers for providing this great opportunity and for the project guide as well 👏
+- Huge appriciation to [Gregoire Vella](https://www.behance.net/gregoirevella) for providing design system guidelines since this project is built upon that ✨ 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
Great job @arabiu033 🤝

- [ ] [Optional] Kindly fix the broken MIT license link to make your readme file more professional.

Reply:
Thanks, @1GORDON for the review :heart: I will fix that